pints/_mcmc/_rao_blackwell_ac.py

```diff
@@ -0,0 +1,109 @@
+#
+# Rao-Blackwell Adaptive MCMC method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class RaoBlackwellACMC(pints.GlobalAdaptiveCovarianceMC):
+    """
+    Rao-Blackwell adaptive MCMC, as described by Algorithm 3 in [1]_. The
+    algorithnm consists of the following steps::
```

Comment:
`::`, on its own, indicates the start of a code block. But "initialise mu0 and sigma0" is not code

Reply:
Changed